src/NodaTime.Demo/XmlSchemaDemo.XmlCodeExporterAssemblyCreator.cs

```diff
@@ -0,0 +1,111 @@
+using System;
```

Comment:
Could you add copyright statements to all files please? I can do it afterwards if not.

Reply:
Done in 2080028524c3383d9c6cbce852c32bd440ceed22.